docs/CONTRIBUTING.md

```diff
@@ -16,33 +16,32 @@ To get setup, please follow these guidelines:
 8. Under "Scopes", select "bot"
 9. Permissions. Enable only these permissions:
 
--   Manage Server
 -   Manage Roles
 -   Manage Channels
 -   Kick Members
 -   Ban Members
--   Manage Emojis
+-   Manage Emojis & Stickers
 -   Manage Webhooks
 -   View Channels
 -   All of the permissions under "Text Permissions"
--   No permissions under "Voice Permissions"
+-   Only permissions under "Voice Permissions" are Connect & Use Voice Activity.
 
 10. Select and copy the generated URL under "Scopes." Make sure the value
-    of permissions in the URL reads 4027055222
+    of permissions in the URL reads 536637602902.
```

Comment:
since this value can change based on needs, can we have a link here to the PERMISSIONS file in docs?

Reply:
great idea 💡

Reply:
updated ✅